services/blockchain-coordinator/shared/scheduler.js

```diff
@@ -0,0 +1,129 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2022 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const Queue = require('bull');
+
+const {
+	Logger,
+} = require('lisk-service-framework');
+
+const logger = Logger();
+
+const {
+	isGenesisBlockIndexed,
+	isGenesisAccountsIndexed,
+	getDelegateAccounts,
+	getGenesisAccountAddresses,
+	getMissingblocks,
+	getCurrentHeight,
+	getGenesisHeight,
+} = require('./sources/indexer');
+
+const {
+	getEnabledModules,
+} = require('./sources/connector');
+
+const config = require('../config');
+const Signals = require('./signals');
+
+const blockIndexQueue = new Queue('Blocks', config.endpoints.redis, {
+	defaultJobOptions: config.queue.defaultJobOptions,
+});
+
+const accountIndexQueue = new Queue('Accounts', config.endpoints.redis, {
+	defaultJobOptions: config.queue.defaultJobOptions,
+});
```

Comment:
Kindly update the config to use the same properties and environment variables across both `coordinator` and `indexer` microservices. Also, please update the config for the queue name as constants in both the microservices.

```suggestion
const blockIndexQueue = new Queue('Blocks', config.endpoints.messageQueue, {
	defaultJobOptions: config.queue.defaultJobOptions,
});

const accountIndexQueue = new Queue('Accounts', config.endpoints.messageQueue, {
	defaultJobOptions: config.queue.defaultJobOptions,
});
```